### PR TITLE
Issue #2693 - Updated parameter units for consistency

### PR DESCRIPTION
**Pull request by danmergens**
- updated RELEASE_NOTES with version update
- removed all instances where description was redefining Units
- corrected all usage of micro prefix to be 'µ'
- corrected two entries that were split on multiple lines

### CODE REVIEW

**Comment by danmergens** [comment]
If this looks good and you accept the pull request, it should automatically push it to your current pull request to the master repo. If it doesn't, you should be able to open your pull request (https://github.com/oceanobservatories/preload-database/pull/51) and there should be an update option there.

File: csv/ParameterDefs.csv [hunk 1]
```diff
@@ -2116,23 +2116,23 @@ OPTAA_DJ_CSPP,OPTAA_DJ_CSPP,on_seconds,PD2611,quantity,,float32,,s,-9999999,Powe
...
 THSPH_A,THSPH,thsph_ph_noref_acl,PD2633,function,,float32,,1,-9999,Vent Fluid pH - Assumed Chloride - No Reference AgCl,0,,PFID113,"{""counts_ysz"":""PD2260"",""temperature"":""PD2624"",""arr_agclref"":""CC_arr_agclref"",""e2l_ysz"":""CC_e2l_ysz"",""arr_hgo"":""CC_arr_hgo"",""arr_agcl"":""CC_arr_agcl"",""arr_tac"":""CC_arr_tac"",""arr_tbc1"":""CC_arr_tbc1"",""arr_tbc2"":""CC_arr_tbc2"",""arr_tbc3"":""CC_arr_tbc3""}",THSPHPH-PH-NOREF-ACL,,THSPHPH-PH-NOREF-ACL_L2,"In-situ pH at high temperature location, using an assumed [Cl-] from a physical sample and a derived value for the AgCl electrode. Unitless.",,Science Data,L2
```
Comment (leilabbb) [inline csv/ParameterDefs.csv]: remove the word Unitless from the description

File: csv/ParameterDefs.csv [hunk 2]
```diff
@@ -2216,61 +2216,61 @@ PHSEN,PHSEN,ph_light_measurements,PD2708,array<quantity>,,int16,,counts,-9999,Ar
...
+WAVSS_A_DCL,WAVSS-A,peak_wave_period,PD2782,quantity,,float32,,sec,-9999999,Wave Period - Peak,4,,,,,,WAVSTAT-TP_L0,Peak wave period; period where wave spectrum has its maximum.,Peak Wave Period,Unprocessed Data,L0
+WAVSS_A_DCL,WAVSS-A,wave_period_tp5,PD2783,quantity,,float32,,sec,-9999999,Wave Period - Read Method,4,,,,,,WAVSTAT-TP5_L0,Peak wave period computed via Read method; an alternative to TP.,Peak Wave Period - Read Method,Unprocessed Data,L0
+WAVSS_A_DCL,WAVSS-A,wave_height_hmo,PD2784,quantity,,float32,,m,-9999999,Wave Height - HMO,4,,,,,,WAVSTAT-HMO_L0,Significant wave height estimated by an alternate method based on spectral moments (HMO).,Significant Wave Height Estimated HMO,Unprocessed Data,L0
+WAVSS_A_DCL,WAVSS-A,mean_direction,PD2785,quantity,,float32,,degrees,-9999999,Mean Direction of Wave Field,4,,,,,,WAVSTAT-D_L0,Mean direction of wave field against true north.,Mean Direction of Wave Field,Unprocessed Data,L0
 WAVSS_A_DCL,WAVSS-A,mean_spread,PD2786,quantity,,float32,,degrees,-9999999,Mean Spread,4,,,,,,,Mean directional spread of wave field (units: degrees).,,,
```
Comment (leilabbb) [inline csv/ParameterDefs.csv]: units was not removed from the description here.

File: csv/ParameterDefs.csv [hunk 3]
```diff
@@ -2116,23 +2116,23 @@ OPTAA_DJ_CSPP,OPTAA_DJ_CSPP,on_seconds,PD2611,quantity,,float32,,s,-9999999,Powe
...
 THSPH_A,THSPH,thsph_ph_noref,PD2632,function,,float32,,1,-9999,Vent Fluid pH - Measured Chloride - No Reference AgCl,0,,PFID112,"{""counts_ysz"":""PD2260"",""temperature"":""PD2624"",""arr_agclref"":""CC_arr_agclref"",""e2l_ysz"":""CC_e2l_ysz"",""arr_hgo"":""CC_arr_hgo"",""arr_agcl"":""CC_arr_agcl"",""arr_tac"":""CC_arr_tac"",""arr_tbc1"":""CC_arr_tbc1"",""arr_tbc2"":""CC_arr_tbc2"",""arr_tbc3"":""CC_arr_tbc3"",""chl"":""PD966""}",THSPHPH-PH-NOREF,,THSPHPH-PH-NOREF_L2,"In-situ pH at high temperature location, using measured [Cl-] from TRHPH and a derived value for the AgCl electrode. Unitless.",,Science Data,L2
```
Comment (leilabbb) [inline csv/ParameterDefs.csv]: remove the word Unitless from the description

File: csv/ParameterDefs.csv [hunk 4]
```diff
@@ -2116,23 +2116,23 @@ OPTAA_DJ_CSPP,OPTAA_DJ_CSPP,on_seconds,PD2611,quantity,,float32,,s,-9999999,Powe
...
 THSPH_A,THSPH,thsph_ph_acl,PD2631,function,,float32,,1,-9999,Vent Fluid pH - Assumed Chloride,0,,PFID111,"{""counts_ysz"":""PD2260"",""counts_agcl"":""PD2261"",""temperature"":""PD2624"",""e2l_ysz"":""CC_e2l_ysz"",""e2l_agcl"":""CC_e2l_agcl"",""arr_hgo"":""CC_arr_hgo"",""arr_agcl"":""CC_arr_agcl"",""arr_tac"":""CC_arr_tac"",""arr_tbc1"":""CC_arr_tbc1"",""arr_tbc2"":""CC_arr_tbc2"",""arr_tbc3"":""CC_arr_tbc3""}",THSPHPH-PH-ACL,,THSPHPH-PH-ACL_L2,"In-situ pH at high temperature location, using assumed [Cl-] from a physical sample and a working reference AgCl electrode. Unitless.",,Science Data,L2
```
Comment (leilabbb) [inline csv/ParameterDefs.csv]: remove the word Unitless from the description

File: csv/ParameterDefs.csv [hunk 5]
```diff
@@ -2116,23 +2116,23 @@ OPTAA_DJ_CSPP,OPTAA_DJ_CSPP,on_seconds,PD2611,quantity,,float32,,s,-9999999,Powe
...
+THSPH_A,THSPH,thsph_temp_ref,PD2629,function,,float32,,ºC,-9999999,Reference Thermistor Temperature,2,,PFID104,"{""ts_rawdec_r"":""PD2266"",""e2l_r"":""CC_e2l_r"",""l2s_r"":""CC_l2s_r""}",THSPHTE-REF,,THSPHTE-REF_L1,Reference Thermistor Temperature at the electrical cold-junction in the sensor wand.,Reference Thermistor Temperature,,L1
 THSPH_A,THSPH,thsph_ph,PD2630,function,,float32,,1,-9999,Vent Fluid pH - Measured Chloride,0,,PFID110,"{""counts_ysz"":""PD2260"",""counts_agcl"":""PD2261"",""temperature"":""PD2624"",""e2l_ysz"":""CC_e2l_ysz"",""e2l_agcl"":""CC_e2l_agcl"",""arr_hgo"":""CC_arr_hgo"",""arr_agcl"":""CC_arr_agcl"",""arr_tac"":""CC_arr_tac"",""arr_tbc1"":""CC_arr_tbc1"",""arr_tbc2"":""CC_arr_tbc2"",""arr_tbc3"":""CC_arr_tbc3"",""chl"":""PD966""}",THSPHPH-PH,,THSPHPH-PH_L2,"In-situ pH at high temperature location, using measured [Cl-] from TRHPH and a working reference AgCl electrode. Unitless.",,Science Data,L2
```
Comment (leilabbb) [inline csv/ParameterDefs.csv]: remove the word Unitless from the description

**Pull request merged**